AlfrescoApp/View Controllers/Document Preview View Controller/DocumentPreviewViewController.m

```diff
@@ -330,6 +331,15 @@ - (void)documentUpdated:(NSNotification *)notification
     }
 }
 
+- (void)documentSynced:(NSNotification *)notification
+{
```

Comment:
Please check that the document received in the notification is the same document as the document currently being displayed in the preview.